contributing-device-support.md

```diff
@@ -1,120 +1,207 @@
+and a number of directories out of which the mandatory ones are:
 
-### meta-resin-`<board-family>` breakout
-----------------------------------------
+- `conf` directory - contains the following files:
+    - `layer.conf`, see the [layer.conf](https://github.com/resin-os/resin-raspberrypi/blob/master/layers/meta-resin-raspberrypi/conf/layer.conf) from `meta-resin-raspberrypi` for an example, and see [Yocto documentation](http://www.yoctoproject.org/docs/2.0/mega-manual/mega-manual.html#bsp-filelayout-layer)
+    - `samples/bblayers.conf.sample` file in which all the required Yocto layers are listed, see this [bblayers.conf.sample](https://github.com/resin-os/resin-raspberrypi/blob/master/layers/meta-resin-raspberrypi/conf/samples/bblayers.conf.sample) from `meta-resin-raspberrypi` for an example, and see the [Yocto documentation](http://www.yoctoproject.org/docs/2.0/mega-manual/mega-manual.html#var-BBLAYERS)
+    - `samples/local.conf.sample` file which defines part of the build configuration (see the meta-resin [README.md][meta-resin-readme] for an overview of some of the variables use in the `local.conf.sample` file). An existing file can be used (e.g. [local.conf.sample](https://github.com/resin-os/resin-raspberrypi/blob/master/layers/meta-resin-raspberrypi/conf/samples/local.conf.sample)) but making sure the "Supported machines" area lists the appropriate machines this repository is used for. See also the [Yocto documentation](http://www.yoctoproject.org/docs/2.0/mega-manual/mega-manual.html#structure-build-conf-local.conf).
 
-This directory contains a COPYING.Apache-2.0 file with the Apache Version 2.0 license, a README.md file specifying the supported boards and a number of directories out of which the mandatory ones are:
+- `recipes-containers/docker-disk` directory, whichcontains docker-resin-supervisor-disk.bbappend that shall define the following variable(s):
 
-- conf directory - contains the following files:
+    - `SUPERVISOR_REPOSITORY_<yocto-machine-name>`: this variable is used to specify the build of the supervisor. It can be one of (must match the architecture of the board):
+        *  **resin/armv7hf-supervisor** (for armv7 boards),
+        * **resin/i386-supervisor**
+        (for x86 boards),
+        * **resin/amd64-supervisor** (for x86-64 boards),
+        * **resin/rpi-supervisor** (for raspberry pi 1),
+        * **resin/armel-supervisor** (for armv5 boards).
 
-    - layer.conf (see this [layer.conf][layer.conf intel] for an example)
+    - `LED_FILE_<yocto-machine-name>`: this variable should point to the [Linux sysfs path of an unused LED](https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-class-led) if available for that particular board. This allows the unused LED to be flashed for quick visual device identification purposes). If no such unused LED exists, this variable shall not be used.
```

Comment:
"visual device identification purposes)" - probably remove the closing bracket here